### DISCUSSION TITLE
Can I move my Actions beta participation from my user to my organization(s)?

### DISCUSSION THREAD

**Discussion by sejr** [community/community#26110] [Actions]
<p>Title says it all. When I signed up for the beta, I didn’t notice the option to choose the account to be waitlisted. The repos I want to test are not owned by my personal user, unfortunately.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>If you haven’t already, please contact us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> with the info and we’ll see what we can do.</p>

**Comment by sejr** [comment]
<p>Just did – thank you!</p>

**Comment by chrispat** [comment]
<p>What organization would you like enabled?</p>

**Comment by sejr** [comment]
<p><a href="https://github.com/vitahq" rel="nofollow noopener">https://github.com/vitahq</a></p>

**Comment by chrispat** [comment]
<p>You should be enabled now</p>
